packages/web/src/services/audius-sdk/audiusSdk.ts

```diff
@@ -43,7 +43,12 @@ const initSdk = async () => {
   const discoveryNodeSelector = await discoveryNodeSelectorService.getInstance()
 
   const message = `signature:${new Date().getTime()}`
-  const signature = await audiusWalletClient.signMessage({ message })
+  const signature = await audiusWalletClient
+    .signMessage({ message })
+    .catch((e) => {
+      console.warn(e)
```

Comment:
probably should not be a warn. will make for a really messy console

Reply:
Yeah fwiw this was a quick and dirty impl. I think doing this "right" would be throwing a specific error type and filtering for that specific error before swallowing it (w/o log)